hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieRealtimePath.java

```diff
@@ -89,7 +89,7 @@ public boolean getBelongsToIncrementalQuery() {
   }
 
   public boolean isSplitable() {
-    return !toString().contains(".log") && !includeBootstrapFilePath();
+    return !toString().contains(".log") && !deltaLogFiles.isEmpty() && !includeBootstrapFilePath();
   }
```

Comment:
empty or not empty ?

Reply:
Thanks, just find it and fixed! add more details to describe the issue. It's safe to review now